toolbox

```diff
@@ -1992,8 +1950,14 @@ case $op in
             fi
         fi
 
-        $ls_images && [ "$images" != "" ] && echo "$images"
-        $ls_containers && [ "$containers" != "" ] && echo "$containers"
```

Comment:
Good catch! I merged this patch via https://github.com/debarshiray/toolbox/pull/258 since it's obviously correct. I tweaked the commit message to make the problem more obvious.